substratevm/src/com.oracle.svm.methodhandles/src/com/oracle/svm/methodhandles/MethodHandleFeature.java

```diff
@@ -90,8 +105,41 @@ public boolean isInConfiguration(IsInConfigurationAccess access) {
 
     @Override
     public void duringSetup(DuringSetupAccess access) {
+        seenMethodHandles = ConcurrentHashMap.newKeySet();
         directMethodHandleClass = access.findClassByName("java.lang.invoke.DirectMethodHandle");
         boundMethodHandleClass = access.findClassByName("java.lang.invoke.BoundMethodHandle");
+        delegatingMethodHandleClass = access.findClassByName("java.lang.invoke.DelegatingMethodHandle");
+        getDelegatingMethodHandleTarget = ReflectionUtil.lookupMethod(delegatingMethodHandleClass, "getTarget");
+        getDelegatingMethodHandleTarget.setAccessible(true);
```

Comment:
`lookupMethod` and `lookupField` already perform the `setAccessible`

Reply:
Thanks for the catch! Fixed.